### ISSUE TITLE
Cannot import records in Classic UI

### ISSUE THREAD

**Issue by shunkosa** [xgeek-net/salesforce-smart-importer#1]
Hello, 

I installed this extension and tried the sample csv file in the repository but spinner sticks and couldn't proceed to field mapping. It occurred in listview, record details, and setting home. Do I need additional settings? 
<img width="596" alt="smart_importer" src="https://user-images.githubusercontent.com/1404346/41021000-dc0a990a-699f-11e8-972b-4298af80a990.png">

**Comment by xgeek-net** [comment]
Hi, @skosaka 

Which edition are you using? 
And could you check the below two settings?
```
In Classic, don not check on “Require HttpOnly attribute” option in “Session Settings”. 
If you login with a Non-Admin user, make sure “API Enabled” in profile has been checked.
```

**Comment by shunkosa** [comment]
Hi @xgeek-net
Thank you for your response. I tried in DE org. I checked that the session setting and the permission (system admin profile) are correct. Let me know if you need more information.

**Comment by xgeek-net** [comment]
Hi, @skosaka 

A similar issue (bug on WinOS) has been reproduced and fixed at v0.1.3, could you try it again? ( You can download sample csv from [here](https://github.com/xgeek-net/salesforce-smart-importer/tree/master/Sample)
If it still doesn't work, click "Report Issue on Github" button with process log as below, the log will be helpful for issue reproduction.

![2018-06-07 11 45 04](https://user-images.githubusercontent.com/5466487/41075590-445c8358-6a48-11e8-812b-048a336d1ffc.png)

**Comment by shunkosa** [comment]
Hello, @xgeek-net 
I re-installed and it worked well! Thank you for your support :)
